src/main/resources/the_warlordResources/localization/eng/WarlordMod-Card-Strings.json

```diff
@@ -1,11 +1,38 @@
 {
-  "the_warlord:Strike_Warlord": {
+  "the_warlord:debu_appbleed_atk": {
+    "NAME": "debu_appbleed_atk",
+    "DESCRIPTION": "plc NL Deal !D! damage NL Apply 1 the_warlord:Bleed."
+  },
+
+  "the_warlord:debu_appgash_atk": {
+    "NAME": "debu_appgash_atk",
+    "DESCRIPTION": "plc NL Deal !D! damage NL Apply 1 the_warlord:Gash."
+  },
+
+  "the_warlord:Strike": {
     "NAME": "Strike",
     "DESCRIPTION": "Deal !D! damage."
   },
-  "the_warlord:Defend_Warlord": {
+  "the_warlord:Defend": {
     "NAME": "Defend",
     "DESCRIPTION": "Gain !B! block."
-  }
+  },
+  "the_warlord:Ration": {
+    "NAME": "Ration",
+    "DESCRIPTION": "Draw a card. NL Next turn, add a Syrette to your hand.",
+    "UPGRADE_DESCRIPTON": "Draw a card. NL Next turn, add a Syrette to your hand."
```

Comment:
It seems like you haven't pushed you stealth fix for this yet?

Reply:
[https://github.com/Cartopol/STS_Community_Warlord_Mod/pull/1/commits/57381a71a5ec85dee6859b4bb018f14e65089f45](The fix was in another commit I pushed shortly after)